package.json

```diff
@@ -77,14 +85,15 @@
     "ts-node": "^3.2.0",
     "tslint": "^4.5.1",
     "tslint-microsoft-contrib": "^4.0.1",
-    "typescript": "2.4.1",
+    "typescript": "~2.3.2",
```

Comment:
I’ve got a PR open to reduce the number of warnings when using 2.4.x to 1: https://github.com/eslint/typescript-eslint-parser/pull/347; but otherwise, the new version of TS seems to be compatible.